eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -182,13 +184,16 @@ instance EraPParams era => ToCBOR (ConwayGovPredFailure era) where
 instance EraPParams era => FromCBOR (ConwayGovPredFailure era) where
   fromCBOR = fromEraCBOR @era
 
+data ConwayGovEvent era 
```

Comment:
And addition of this type in the changelog would also be appreciated :slightly_smiling_face: